cmake/linker_script/arm/linker.cmake

```diff
@@ -149,7 +149,7 @@ zephyr_linker_section(NAME .ramfunc GROUP RAM_REGION SUBALIGN 8)
 if(CONFIG_USERSPACE)
   # This is where the app_mem_partition stuff is going to be placed, once it
   # is generated by gen_app_partitions.py. _app_smem has its own init-copy
-  # handling in z_data_copy, so put it in RAM_REGIOM rather than DATA_REGION
+  # handling in arch_data_copy, so put it in RAM_REGIOM rather than DATA_REGION
```

Comment:
There is an existing typo: RAM_REGIOM -> RAM_REGION

Reply:
well, not part of my change, but fixed it anyways :)